dashboard/app/views/levels/_free_response.html.haml

```diff
@@ -1,16 +1,33 @@
-:javascript
-  function getResult() {
-    return {
-      response: $('.response').val(),
-      result: true
-    };
-  }
+- level ||= @level
+- last_attempt ||= @last_attempt
+- index ||= nil
+- number ||= nil
+
+- if index
```

Comment:
I passed in a variable called standalone in the other cases to differentiate whether it's a standalone level or one embedded in a LevelGroup.